Unable to build on Big Sur using 3.9

`Collecting install
  Downloading install-1.3.4-py3-none-any.whl (3.1 kB)
Collecting amazon-transcribe
  Using cached amazon_transcribe-0.2.0-py2.py3-none-any.whl (36 kB)
Collecting awscrt==0.6.1
  Using cached awscrt-0.6.1.tar.gz (2.1 MB)
Using legacy 'setup.py install' for awscrt, since package 'wheel' is not installed.
Installing collected packages: install, awscrt, amazon-transcribe
    Running setup.py install for awscrt ... error
    ERROR: Command errored out with exit status 1:
     command: /Users/rdigiorgio/PycharmProjects/pythonProject/p3/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/gq/fj424fsx04g4_tlmbs0361j80000gq/T/pip-install-968jhbdj/awscrt/setup.py'"'"'; __file__='"'"'/private/var/folders/gq/fj424fsx04g4_tlmbs0361j80000gq/T/pip-install-968jhbdj/awscrt/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /private/var/folders/gq/fj424fsx04g4_tlmbs0361j80000gq/T/pip-record-mbdp5841/install-record.txt --single-version-externally-managed --compile --install-headers /Users/rdigiorgio/PycharmProjects/pythonProject/p3/include/site/python3.9/awscrt`


First Error

`   clang -Wno-unused-result -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -I/usr/local/include -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk -I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include -I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/Tk.framework/Versions/8.5/Headers -I/usr/local/include -I/usr/local/opt/openssl@1.1/include -I/usr/local/opt/sqlite/include -I/Users/rdigiorgio/PycharmProjects/pythonProject/p3/include -I/usr/local/Cellar/python@3.9/3.9.1_3/Frameworks/Python.framework/Versions/3.9/include/python3.9 -I/private/var/folders/gq/fj424fsx04g4_tlmbs0361j80000gq/T/pip-install-bfbysohb/awscrt/build/deps/install/include -c source/module.c -o build/temp.macosx-11-x86_64-3.9/source/module.o -Wextra -Werror -Wno-strict-aliasing -std=gnu99
    source/module.c:580:10: error: 'PyEval_ThreadsInitialized' is deprecated [-Werror,-Wdeprecated-declarations]
        if (!PyEval_ThreadsInitialized()) {
             ^
    /usr/local/Cellar/python@3.9/3.9.1_3/Frameworks/Python.framework/Versions/3.9/include/python3.9/ceval.h:129:1: note: 'PyEval_ThreadsInitialized' has been explicitly marked deprecated here
    Py_DEPRECATED(3.9) PyAPI_FUNC(int) PyEval_ThreadsInitialized(void);
    ^
`

--- a/CHANGES.rst
+++ b/CHANGES.rst
@@ -1,6 +1,11 @@
 Changes
 =======
 
+dev
+---
+
+* The required version of awscrt has been upgraded to 0.11.11
+
 0.2.0 (2020-11-13)
 -----------------
 

--- a/amazon_transcribe/signer.py
+++ b/amazon_transcribe/signer.py
@@ -20,7 +20,7 @@ from awscrt.auth import (
     AwsSigningAlgorithm,
     AwsSigningConfig,
     AwsSignatureType,
-    AwsSignedBodyValueType,
+    AwsSignedBodyValue,
     AwsSignedBodyHeaderType,
     aws_sign_request,
 )
@@ -58,7 +58,7 @@ class RequestSigner:
             credentials_provider=credential_provider,
             region=self.region,
             service=self.service_name,
-            signed_body_value_type=AwsSignedBodyValueType.EMPTY,
+            signed_body_value=AwsSignedBodyValue.UNSIGNED_PAYLOAD,
             signed_body_header_type=AwsSignedBodyHeaderType.NONE,
         )
         crt_request = _convert_request(request)

--- a/setup.cfg
+++ b/setup.cfg
@@ -3,7 +3,7 @@ universal = 1
 
 [metadata]
 requires_dist =
-    # TODO
+    awscrt==0.11.11
 
 [flake8]
 # We ignore E203, E501 for this project due to black

--- a/setup.py
+++ b/setup.py
@@ -21,7 +21,7 @@ def find_version(*file_paths):
     raise RuntimeError("Unable to find version string.")
 
 
-requires = ["awscrt==0.6.1"]
+requires = ["awscrt==0.11.11"]
 
 setup(
     name="amazon-transcribe",

